generated/billing/microsoft.billing/2017-02-27-preview/types.md

```diff
@@ -0,0 +1,2 @@
+# Microsoft.Billing @ 2017-02-27-preview
+
```

Comment:
Microsoft.Billing defines two paths that are clashing in the Bicep generator, `/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/{invoiceName}` and `/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/latest`. Bicep should probably be catching this situation and collapsing the overlapping resources into a single definition.